.circleci/config.yml

```
@@ -1,6 +1,9 @@
 # NOTE: Current plan gives 1500 build minutes per month.
 version: 2.1
 
+orbs:
+  go: circleci/go@1.5.0
```

```suggestion
  go: circleci/go@1.15.0
```
Could this be `1.15.0` 🤔 ?

---

No, this is the orb version, not the Go version.

https://github.com/CircleCI-Public/go-orb/releases